{"schemaVersion":{"major":0,"minor":3,"patch":0},"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"tokens":[{"kind":"keyword","text":"@property"},{"kind":"text","text":" ("},{"kind":"keyword","text":"nonatomic"},{"kind":"text","text":", "},{"kind":"keyword","text":"strong"},{"kind":"text","text":", "},{"kind":"keyword","text":"readonly"},{"kind":"text","text":") "},{"kind":"typeIdentifier","text":"NSString","preciseIdentifier":"c:objc(cs)NSString"},{"kind":"text","text":" * "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":";"}],"languages":["occ"]}],"kind":"declarations"}],"abstract":[{"text":"The reason for the routing failure","type":"text"}],"variants":[{"traits":[{"interfaceLanguage":"occ"}],"paths":["\/documentation\/smiclientcore\/smiroutingresult\/failurereason"]}],"kind":"symbol","metadata":{"externalID":"c:objc(pl)SMIRoutingResult(py)failureReason","modules":[{"name":"SMIClientCore"}],"title":"failureReason","role":"symbol","fragments":[{"text":"failureReason","kind":"identifier"}],"roleHeading":"Instance Property","navigatorTitle":[{"kind":"identifier","text":"failureReason"}],"symbolKind":"property"},"sections":[],"identifier":{"url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIRoutingResult\/failureReason","interfaceLanguage":"occ"},"hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIRoutingResult"]]},"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","kind":"symbol","type":"topic","title":"SMIClientCore","url":"\/documentation\/smiclientcore","role":"collection","abstract":[]},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIRoutingResult":{"fragments":[{"kind":"identifier","text":"SMIRoutingResult"}],"title":"SMIRoutingResult","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIRoutingResult","abstract":[{"text":"Represents a routing result within an ","type":"text"},{"code":"SMIConversationEntry","type":"codeVoice"},{"text":".","type":"text"}],"type":"topic","url":"\/documentation\/smiclientcore\/smiroutingresult","kind":"symbol","role":"symbol"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIRoutingResult/failureReason":{"abstract":[{"text":"The reason for the routing failure","type":"text"}],"identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIRoutingResult\/failureReason","role":"symbol","fragments":[{"text":"failureReason","kind":"identifier"}],"title":"failureReason","kind":"symbol","type":"topic","url":"\/documentation\/smiclientcore\/smiroutingresult\/failurereason"}}}